src/pages/generate/pages/result/components/DetectionHotspots.tsx

```
@@ -155,51 +119,12 @@ const DetectionHotspots = ({
-        if (!categoryId) return null;
-        return { hotspot, resolvedCode };
-      })
-      .filter((item): item is DisplayHotspot => Boolean(item));
-  }, [hotspots, allowedCategories, detectedCodeToCategoryId]);
-
-  const hasHotspots = displayHotspots.length > 0;
+  const { imgRef, containerRef, hotspots, error } = useFurnitureHotspots(
+    imageUrl,
+    mirrored,
+    shouldInferHotspots,
+    hotspotOptions
```

ESLint

[error] Insert `,`

(prettier/prettier)